src/leaflike/bookmarks.clj

```diff
@@ -30,26 +27,27 @@
   [{:keys [params] :as request}]
   (let [bookmark (-> (select-keys params [:title :url :tags])
                      format-tags)
-        user    (get-user request)]
+        user    (hutils/get-user request)
+        tags (:tags bookmark)]
     (if (valid-bookmark? bookmark)
-      (let [bookmark (assoc bookmark
-                            :member_id (:id user)
-                            :created_at (utils/get-timestamp))]
-        (bm-db/create bookmark)
+      (let [bookmark (-> bookmark
+                         (dissoc :tags)
+                         (assoc :member_id (:id user)
+                                :created_at (utils/get-timestamp)))]
+        (let [bookmark-id (bm-db/create bookmark)]
```

Comment:
@samrat This can be pulled into the enclosing `let` making this `let` redundant

Reply:
Fixed